stdlib/public/core/Runtime.swift.gyb

```diff
@@ -496,12 +496,12 @@ class _SwiftNativeNSSet {}
 class _SwiftNativeNSEnumerator {}
 
 @objc @_swift_native_objc_runtime_base(_SwiftNativeNSDataBase)
-public class _SwiftNativeNSData {
+open class _SwiftNativeNSData {
   public init() {}
 }
 
 @objc @_swift_native_objc_runtime_base(_SwiftNativeNSCharacterSetBase)
-public class _SwiftNativeNSCharacterSet {
+open class _SwiftNativeNSCharacterSet {
```

Comment:
They are subclassed in the Foundation module.  I would prefer to move them from the core library into the Foundation overlay, but that is a separate project.

Could you add a comment `// FIXME(ABI): move into the Foundation overlay and remove 'open'.` to *each* one?